src/js/hca/containers/ConfirmationPage.jsx

```diff
@@ -22,33 +22,49 @@ export class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const time = _.get('form.submission.response.timestamp', this.props);
+    const { form } = this.props;
+    const { response } = form.submission;
+    const time = _.get('timestamp', response);
+    const name = form.data.veteranFullName;
+    const healthEligibilityCenter = [
+      'Health Eligibility Center',
+      '2957 Clairmont Rd., Suite 200',
+      'Atlanta, GA 30329'
+    ];
 
     return (
       <div>
-        <div className="usa-width-one-sixth medium-2 columns">
-          <i className="fa fa-check-circle hca-success-icon"></i>
-        </div>
-        <div className="usa-width-five-sixths medium-10 columns">
-          <h4 className="success-copy">You have successfully submitted your application for health care.</h4>
+        <h3 className="confirmation-page-title">Claim received</h3>
+        <p>Normally processed within <strong>a week</strong>.</p>
+        <p>
+          We may contact you for more information or documents.<br/>
+          <i>Please print this page for your records.</i>
+        </p>
+        <div className="inset">
+          <h4>Health Care Benefit Claim <span className="additional">(Form 10-10EZ)</span></h4>
+          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
+
+          <ul className="claim-list">
+            <li>
+              <strong>Date received</strong><br/>
+              <span>{moment(time).format('MMM D, YYYY')}</span>
+            </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.formSubmissionId}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong>
+              <address className="schemaform-address-view">{healthEligibilityCenter.map((line, index) =>
```

Comment:
I don't think we should use an `address` tag for this, since that is meant to be used for the contact information of the owner of the article/page (which is us, not the eligibility center). 

http://html5doctor.com/the-address-element/

Reply:
Thanks @jbalboni, changed.